blocks/commerce-cart/commerce-cart.js

```diff
@@ -61,6 +67,8 @@ export default async function decorate(block) {
   block.innerHTML = '';
   block.appendChild(fragment);
 
+  let isShownUpdateNotification = false;
```

Comment:
As we discussed, for this PR, let's check to see if the cart page URL has an `itemUid` parameter. If it does, it means the user has updated the configurable product options, and we display the message in the cart. If it doesn't, we don't display the message.  

I think this line currently wants to display the message every time. Let's update this line to check for the itemUid.